assets/sass/patterns/molecules/main-menu.scss

```diff
@@ -23,12 +23,12 @@
   text-align: left;
 
   @media only screen and (min-width: #{get-rem-from-px($bkpt-site--wide)}em) {
-    &.main-menu__list_change {
+    &.hidden-wide {
```

Comment:
The reason we don't have a prefix is because I wish for them to be used in the `Link` ViewModel which has no idea of the context.